zonemaster/zonemaster-backend
# Clean up

## Purpose

This PR cleans up various things. Nothing critical, just nice.

## Context

N/A

## Changes

* Fixes indentation and ordering of dependencies in Makefile.PL.
* Makes git ignore MO files.
* Makes `make manifest` not register the dist file for inclusion in the dist file.

## How to test this PR

Verify that `git` ignores MO files:
```sh
(cd share ; make sv.po)
git status
```

Verify that `make manifest` doesn't add the dist file to MANIFEST:
```sh
make dist
make manifest
cat MANIFEST
```
